### DISCUSSION TITLE
x-transition seems non-responsive in the latest version of Alpine.js (3.2.3)

### DISCUSSION THREAD

**Discussion by hchiam** [alpinejs/alpine#1945] [7. Bugs (archived)]
😢 3.2.3: https://codepen.io/hchiam/pen/XWRGgBd seems immediate.

😃 3.2.2: https://codepen.io/hchiam/pen/wvdOeXR transitions nicely by default.

**Comment by KevinBatdorf** [comment]
The fix is in the main branch already so it will be in the next release.
